scripts/cluster_buildings.py

```diff
@@ -95,25 +96,28 @@ def get_central_points_geojson_with_buildings(
     """
     # Classify and process the buildings
     microgrid_buildings = buildings_classification(input_filepath, crs)
-    # Prepare GeoDataFrames and DataFrames to accumulate results
+
+    # Prepare accumulators
     all_central_features = gpd.GeoDataFrame(
         columns=["geometry", "cluster", "name_microgrid"]
     )
     all_microgrid_buildings = gpd.GeoDataFrame(columns=microgrid_buildings.columns)
```

Comment:
if you know in advance that the crs is EPSG by construction, you can add the argument crs=EPSG_BUILDINGS where EPSG_BUILDINGS is a constant value initialized at the beginning of the script.
This should avoid the if else below